internal/oobmigration/migrations/batches/role_assignment_migrator_test.go

```diff
@@ -0,0 +1,96 @@
+package batches
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/keegancsmith/sqlf"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/sourcegraph/log/logtest"
+
+	"github.com/sourcegraph/sourcegraph/internal/database"
+	"github.com/sourcegraph/sourcegraph/internal/database/basestore"
+	"github.com/sourcegraph/sourcegraph/internal/database/dbtest"
+	"github.com/sourcegraph/sourcegraph/internal/types"
+)
+
+func TestRoleAssignmentMigrator(t *testing.T) {
+	ctx := context.Background()
+	logger := logtest.Scoped(t)
+	db := database.NewDB(logger, dbtest.NewDB(logger, t))
+	store := basestore.NewWithHandle(db.Handle())
+
+	migrator := NewRoleAssignmentMigrator(store, 5)
+	progress, err := migrator.Progress(ctx, false)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Nit, but since you use this later in the test when you check the progress, maybe for consistency just:

```suggestion
	assert.NoError(t, err)
```